### PR TITLE
Prm 3121 add can you pay page

### PR DESCRIPTION
**Pull request by RoheelArshad**

### CODE REVIEW

File: app/controllers/findOutHowToAppeal/CanYouPayController.scala [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ * limitations under the License.
+ */
+
+package controllers.findOutHowToAppeal
+
+import config.featureSwitches.{FeatureSwitching, ShowFindOutHowToAppealJourney}
+import config.{AppConfig, ErrorHandler}
+import controllers.predicates.{AuthPredicate, DataRequiredAction, DataRetrievalAction}
+import forms.CanYouPayForm.canYouPayForm
+import forms.DoYouWantToPayNowForm
```
Comment (AndrewRButcher) [inline app/controllers/findOutHowToAppeal/CanYouPayController.scala:23]: Unused import

File: app/controllers/findOutHowToAppeal/CanYouPayController.scala [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          SessionKeys.canYouPay,
+          request.answers)
+        val vatAmount: BigDecimal = 123.45 //TODO Get from session
+        val radioOptions = RadioOptionHelper.radioOptionsForCanYouPayPage(formProvider, CurrencyFormatter.parseBigDecimalToFriendlyValue(vatAmount))
+        Future(Ok(page(formProvider, radioOptions, pageMode)))
+      } else {
+        errorHandler.onClientError(request, NOT_FOUND, "")
+      }
+    }
+  }
```
Comment (AndrewRButcher) [inline app/controllers/findOutHowToAppeal/CanYouPayController.scala:61]: Can you add the routing/navigation and POST endpoint for this page?

File: app/forms/CanYouPayForm.scala [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ */
+
+package forms
+
+import forms.mappings.Mappings
+import play.api.data.Form
+import play.api.data.Forms.single
+
+object CanYouPayForm extends Mappings {
+  final val options = Seq("yes", "no")
```
Comment (AndrewRButcher) [inline app/forms/CanYouPayForm.scala:24]:
```suggestion
  final val options = Seq("yes", "no","paid")
```

File: app/viewtils/RadioOptionHelper.scala [hunk 4]
```diff
@@ -76,6 +76,26 @@ object RadioOptionHelper {
...
+        content = Text(messages("common.radioOption.no.3")),
+        checked = form("value").value.contains("no")
+      ),
+      RadioItem(
+        value = Some("paid"),
+        content = Text(messages("common.radioOption.paid")),
+        checked = form("value").value.contains("paid")
+      )
+    )
+  }
```
Comment (AndrewRButcher) [inline app/viewtils/RadioOptionHelper.scala:97]: Can we get a unit test added for this please?

**Review by AndrewRButcher — CHANGES_REQUESTED** [review]

File: app/utils/SessionKeys.scala [hunk 5]
```diff
@@ -67,6 +67,7 @@ object SessionKeys {
   val fileNames = "fileNames"
   val doYouWantToPayNow = "doYouWantToPayNow"
   val willUserPay = "willUsePay" //TODO: This should be set in the Can You Pay Your VAT Bill page
+  val canYouPay = "canYouPay"
```
Comment (AndrewRButcher) [inline app/utils/SessionKeys.scala:70]: can this sessionKey be removed and use willUserPay instead? or remove the willUserPay key and just canYouPay? seems like we have two keys for the same thing

**Review by AndrewRButcher — CHANGES_REQUESTED** [review]

File: app/utils/SessionKeys.scala (hunk 5 above)
Comment (RoheelArshad) [reply app/utils/SessionKeys.scala:70]: I'll update

**Review by AndrewRButcher — APPROVED** [review]

**Pull request merged**